addons/binding/org.openhab.binding.avmfritz/ESH-INF/thing/thing-types.xml

```diff
@@ -45,126 +45,12 @@
-				<default>15</default>
-			</parameter>
-			<parameter name="asyncTimeout" type="integer" required="false" min="1000" max="60000">
-				<label>Async Timeout</label>
-				<description>Timeout for asynchronous connections.
-				</description>
-				<default>10000</default>
-			</parameter>
-			<parameter name="syncTimeout" type="integer" required="false" min="500" max="15000">
-				<label>Sync Timeout</label>
-				<description>Timeout for synchronous connections.
-				</description>
-				<default>2000</default>
-			</parameter>
-		</config-description>
-    </thing-type>
-
-    <thing-type id="FRITZ_Powerline_1220E_Solo">
-		<supported-bridge-type-refs>
-			<bridge-type-ref id="fritzbox" />
-		</supported-bridge-type-refs>
-        <label>FRITZ!Powerline 1220E</label>
-        <description>FRITZ!Powerline 1220E</description>
-		<channels>
-		</channels>
-		<config-description>
-			<parameter name="ipAddress" type="text" required="true">
-				<context>network-address</context>
-				<label>IP address</label>
-				<description>The local IP address or hostname of the FRITZ!Box
-				</description>
-				<default>fritz.powerline</default>
-			</parameter>
-			<parameter name="port" type="integer" required="false" min="0"
-				max="65335">
-				<label>Port</label>
-				<description>Port of the FRITZ!Box</description>
-			</parameter>
-			<parameter name="protocol" type="text" required="false">
-				<label>Protocol</label>
-				<description>Protocol to connect to the FRITZ!Box (http or https)</description>
-				<default>http</default>
-				<options>
-					<option value="http">HTTP</option>
-					<option value="https">HTTPS</option>
-				</options>
-			</parameter>
-			<parameter name="password" type="text" required="false">
-				<context>password</context>
-				<label>Password</label>
-				<description>Password to access the FRITZ!Box device.
-				</description>
-			</parameter>
-			<parameter name="pollingInterval" type="integer" required="false" min="5" max="60">
-				<label>Polling Interval</label>
-				<description>Interval polling the FRITZ!Box.
-				</description>
-				<default>15</default>
-			</parameter>
-			<parameter name="asyncTimeout" type="integer" required="false" min="1000" max="60000">
-				<label>Async Timeout</label>
-				<description>Timeout for asynchronous connections.
-				</description>
-				<default>10000</default>
-			</parameter>
-			<parameter name="syncTimeout" type="integer" required="false" min="500" max="15000">
-				<label>Sync Timeout</label>
-				<description>Timeout for synchronous connections.
-				</description>
-				<default>2000</default>
-			</parameter>
-		</config-description>
-    </thing-type>
-
-    <thing-type id="FRITZ_Powerline_1000E_Solo">
-		<supported-bridge-type-refs>
-			<bridge-type-ref id="fritzbox" />
-		</supported-bridge-type-refs>
-        <label>FRITZ!Powerline 1000E</label>
-        <description>FRITZ!Powerline 1000E</description>
+        <label>FRITZ!Powerline devices</label>
```

Comment:
This is the label/desc that is shown for ONE device (that is added/discovered/etc.) - so I would suggest to change the label to "FRITZ!Powerline" and the description to "a FRITZ!Powerline device"